internal/api/fetch_files_timing.go

```diff
@@ -0,0 +1,84 @@
+package api
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"slices"
+)
+
+type fetchFilesTimingParams struct {
+	Paths []string `json:"paths"`
+}
+
+type fileTiming struct {
+	Path     string
+	Duration int
```

Comment:
I noticed we store durations as real in the daily file stats table. I'm wondering if int would work?

Reply:
Ah good point. So it's a float with value in second right?
@wooly

Reply:
Resolved

Reply:
Should we change this back to int? I don't think sub-second timing will make a huge amount of difference for SBE, but will simplify our deployment process with a few customers already using the splitter.